components/widgets/popover/index.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Popover as BSPopover} from 'react-bootstrap';
+
+export type Sizes = 'xs' | 'xsmall' | 'sm' | 'small' | 'medium' | 'lg' | 'large';
+
+interface Props {
+    id: string;
+    children?: JSX.Element | JSX.Element[] | string;
+    popoverStyle?: 'info';
+    popoverSize?: Sizes;
+    title?: React.ReactNode;
+    placement?: 'bottom' | 'top' | 'right' | 'left';
+}
+
+const Popover: React.FC<Props> = (props: Props) => (
+    <BSPopover
+        bsStyle={props.popoverStyle}
+        placement={props.placement}
+        bsClass='popover'
+        bsSize={props.popoverSize}
+        {...props}
```

Comment:
Can we be explit here and pass the props one by one (I get that is for code simplification but I think is less clear, and can lead to a bad usage of the API from a javascript library (passing extra arguments directly to the Bootstrap popover)